app/src/main/java/org/fossasia/openevent/app/common/app/binding/BindingAdapters.java

```diff
@@ -42,7 +42,7 @@ public static String floatToStr(Float value) {
     }
 
     public static Long strToLong(String value) {
-        return value == null ?  null : Long.parseLong(value);
+        return (("").equals(value)) ? 0 : Long.parseLong(value);
```

Comment:
Use TextUtils.isEmpty

Secondly, we don't want 0 to go to server if user has not entered anything, we want null